chainquery/compose/docker-compose.yml-prod-example

```diff
@@ -35,14 +36,16 @@ services:
 ## Chainquery ##
 ################
   chainquery:
-    image: lbryio/lbry-docker:linux-x86_64-production
+    build:
+      context: .
+      dockerfile: Dockerfile-linux-x86_64-production
     restart: always
     networks:
       lbry-network:
         ipv4_address: 10.6.1.3
     env_file:
       - env
-      - ../environment/lbrycrd.env
+      - ../environment/chainquery.env
```

Comment:
This is likely fine but I might've been just pulling the lbrycrd env file due to only needing to know the RPC credentials.